### PR TITLE
Update offline bundle instructions from CSI to CSM Operator

### PR DESCRIPTION
**Pull request by KerryKovacevic**
# Description
Updated documentation for offline bundle installation from referencing CSI Operator to CSM Operator.

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|https://github.com/dell/csm/issues/1081|

# Checklist:

- [ ] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

File: dell-csi-helm-installer/csi-offline-bundle.md [hunk 1]
```diff
@@ -2,8 +2,7 @@
 
 ## Description
 
-The `csi-offline-bundle.sh` script can be used to create a package usable for offline installation of the Dell EMC CSI Storage Providers, via either Helm 
-or the Dell CSI Operator. 
+The `csi-offline-bundle.sh` script can be used to create a package usable for offline installation of the Dell EMC CSI Storage Providers, via Helm.
```
Comment (bharathsreekanth) [inline dell-csi-helm-installer/csi-offline-bundle.md:5]: minor edit, change to : 
The `csi-offline-bundle.sh` script can be used to create a package for offline installation of Dell CSI storage providers for deployment via HELM.

File: dell-csi-helm-installer/csi-offline-bundle.md [hunk 2]
```diff
@@ -46,84 +45,93 @@ To perform an offline installation of a driver or the Operator, the following st
 This needs to be performed on a linux system with access to the internet as a git repo will need to be cloned, and container images pulled from public registries.
 
 The build an offline bundle, the following steps are needed:
-1. Perform a `git clone` of the desired repository. For a helm based install, the specific driver repo should be cloned. For an Operator based deployment, the Dell CSI Operator repo should be cloned
-2. Run the `csi-offline-bundle.sh` script with an argument of `-c` in order to create an offline bundle
+1. Perform a `git clone` of the desired repository. For a helm based install, the specific driver repo should be cloned. For an Operator based deployment, the Dell CSM Operator repo should be cloned
```
Comment (bharathsreekanth) [inline dell-csi-helm-installer/csi-offline-bundle.md:48]: Change helm to HELM or Helm (use this same convention everywhere). Sorry, I know some of these are legacy and not really in the scope of this PR, but I think a search and replace might be easy to drive consistency.

**Review by bharathsreekanth — COMMENTED** [review]
I have reviewed powerflex but my comments will apply across all the others.

**Comment by KerryKovacevic** [comment]
> I have reviewed powerflex but my comments will apply across all the others.

Updated, will fix for the rest of the repos as well.

**Review by shanmydell — CHANGES_REQUESTED** [review]
image
Needs to be updated

**Review by bharathsreekanth — DISMISSED** [review]
TY @KerryKovacevic for these changes

**Review by PeresKereotubo — DISMISSED** [review]

**Review by adarsh-dell — APPROVED** [review]

**Review by bharathsreekanth — APPROVED** [review]

**Review by shanmydell — APPROVED** [review]

**Pull request merged**